test/threads_test.py

```diff
@@ -91,8 +108,9 @@ def todo_test_benchmark_workers(self):
         # You can pass in benchmark data, and functions if you want.
         # a_bench_func - f(data)
         # the_data - data to work on.
-
-        self.fail()
+        optimal_workers = threads.benchmark_workers()
+        self.assertTrue(type(optimal_workers) == int)
```

Comment:
instead of `type` use` isinstance(optimal_workers, int)` - otherwise various code linters (like pylint) will complain.

Reply:
Made this change